champselect.py

```
@@ -120,55 +109,57 @@ def do_champ(**kwargs) -> None:
 
     # If current pick intent isn't valid, loop through user's config to find a champ to pick
     options: list[str] = u.get_backup_config_champs(connection.get_assigned_role())
     for pick in options:
-        is_valid: bool = is_valid_pick(pick)
+        is_valid: bool = is_valid_pick(connection, pick)
         if is_valid:
             return pick
     # Last config option isn't valid
     if not is_valid:
-        raise Exception("Unable to find a valid champion to pick.")
+        raise RuntimeError("Unable to find a valid champion to pick.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)